pkg/authenticator/config/config.go

```diff
@@ -35,11 +35,11 @@ const (
 	DefaultConjurVersion = "5"
 
 	// DefaultTokenRefreshTimeout is the default time the system waits to reauthenticate on error
-	DefaultTokenRefreshTimeout = 6 * time.Minute
+	DefaultTokenRefreshTimeout = "6m0s"
```

Comment:
in SP I wrote with `time` was there a particular reason why this was changed?

Reply:
the util receives a string as an input to match the value that will be given in the env. i think that it makes sense that both the value from the env and the default value are of the same type. after we get the value (from the env or default) we convert it to the required type (int, duration, etc.).